defaults/config.js

```diff
@@ -312,6 +304,16 @@ module.exports = {
 	// This value is set to `["sqlite", "text"]` by default.
 	messageStorage: ["sqlite", "text"],
 
+	// ### `sqliteConfig`
+	//
+	// `maxDaysHistory`:
+	// 		Defines the maximum number of days of history to keep in the database.
+	// 		Undefined/-1/0 is treated an unlimited.
```

Comment:
exactly undefined
We should have one way and only one way of configuring something.